docs/learn/vocabulary/flavourisation.mdx

```diff
@@ -49,7 +49,7 @@ When `r` or `ŕ` are preceded by a consonant and not followed by a vowel, they c
 
 The consonant `g` is pronounced more like an aspirated `h` in Ukrainian, Belarussian, Czech, Slovak and Upper Sorbian.
 
-Whereas Standard Interslavic has two softened consonants (`lj` and `nj`), the etymological alphabet has seven: `ĺ ń ŕ t́ d́ ś ź` (note that `lj` and `nj` are identical to `ĺ` and `ń`). The remaining five are largely absent in South Slavic, **`ś` `ź`** also in Czech and Slovak and `ŕ` in Slovak and Belarussian. If you want to give a text a more North-Slavic look-and-feel, it deserves recommendation to include them all. In that case it would of course be logical if all softened consonants were marked the same way: with the soft sign `ь` in Cyrillic, with an acute or a caron (háček) in Latin orthography. Instead of using `ľ` or `ĺ` (both encountered in Slovak only), it is also possible to use `l` for the soft :ipa[ʎ] and `ł` for the hard :ipa[l]. This method is used in Polish, Sorbian and some forms of Belarusian `Łacinka` and Ukrainian `Latynka`.
+Whereas Standard Interslavic has two softened consonants (`lj` and `nj`), the etymological alphabet has seven: `ĺ ń ŕ t́ d́ ś ź` (note that `lj` and `nj` are identical to `ĺ` and `ń`). The remaining five are largely absent in South Slavic, `ś` `ź` also in Czech and Slovak and `ŕ` in Slovak and Belarussian. If you want to give a text a more North-Slavic look-and-feel, it deserves recommendation to include them all. In that case it would of course be logical if all softened consonants were marked the same way: with the soft sign `ь` in Cyrillic, with an acute or a caron (háček) in Latin orthography. Instead of using `ľ` or `ĺ` (both encountered in Slovak only), it is also possible to use `l` for the soft :ipa[ʎ] and `ł` for the hard :ipa[l]. This method is used in Polish, Sorbian and some forms of Belarusian :notr[Łacinka] and Ukrainian :notr[Latynka].
```

Comment:
```suggestion
Whereas Standard Interslavic has two softened consonants (`lj` and `nj`), the etymological alphabet has seven: `ĺ` `ń` `ŕ` `t́` `d́` `ś` `ź` (note that `lj` and `nj` are identical to `ĺ` and `ń`). The remaining five are largely absent in South Slavic, `ś` `ź` also in Czech and Slovak and `ŕ` in Slovak and Belarussian. If you want to give a text a more North-Slavic look-and-feel, it deserves recommendation to include them all. In that case it would of course be logical if all softened consonants were marked the same way: with the soft sign `ь` in Cyrillic, with an acute or a caron (háček) in Latin orthography. Instead of using `ľ` or `ĺ` (both encountered in Slovak only), it is also possible to use `l` for the soft :ipa[ʎ] and `ł` for the hard :ipa[l]. This method is used in Polish, Sorbian and some forms of Belarusian :notr[Łacinka] and Ukrainian :notr[Latynka].
```